### ISSUE TITLE
Bug in documentation (reserve margin)

### ISSUE THREAD

**Issue by FLomb** [calliope-project/calliope#185]
## Problem description

There seems to be a bug in the visualisation of the reserve_margin constraint in the documentation (mathematical formulation section). Probably something went wrong with the coding required by sphinx, which did not translate the code into a mathematical equation.

**Comment by brynpickering** [comment]
Looks fine to me in the [stable](https://calliope.readthedocs.io/en/stable/user/ref_formulation.html) release, could you screenshot it and post image here?

**Comment by FLomb** [comment]
Sorry, my fault: I just realised I was seeing a "non-mathematical" version of the formula only because I had reached it through the "search" button, instead of normal navigation.
So, in order to highlight the words "reserve margin" that I had looked for, I guess that sphinx authomatically displayed the "plain-text/code" version of the formula.
![image](https://user-images.githubusercontent.com/26432077/51327765-52d81e80-1a72-11e9-8198-08b6183a5de2.png)

Should I close the issue?
